independent-publisher-connectors/U.K. Government Bank Holidays/UK-Bank-Holidays.swagger.json

```diff
@@ -0,0 +1,55 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "UK Bank Holidays",
+    "description": "Bank holidays list retrieved from Government web site https://www.gov.uk",
```

Comment:
@martynlesbirel Can you please provide some more information to understand your connector more.

Documentation - [Description of connector](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#description)

Reply:
Description revised, please review.